src/html/index.html

```diff
@@ -0,0 +1,71 @@
+{#
+  Nunjucks Templating!
+  Homepage: https://mozilla.github.io/nunjucks/
+  Docs: https://mozilla.github.io/nunjucks/api.html
+#}
+
+{% extends 'layouts/application.html' %}
+
+{% block content %}
+<div id="top-fold">
+  <div id="title-box">
+  <p id="subtitle">A Viget Exploration</p>
+    <h1 id="title">
+      <a href="/">
+        <img src="images/title.svg" alt="Improving Medical Adherence" />
+      </a>
+    </h1>
+    <img src="images/yellow-pill.svg" alt="Yellow pill." id="yellow-pill">
+    <img src="images/pills.svg" alt="Green and light yellow pills." id="pills">
+    <img src="images/more-pills.svg" alt="Three pill tablets." id="more-pills">
+    <img src="images/purple-pills.svg" alt="Two purple pills." id="purple-pills">
+  </div>
+    <nav class="pillbar">
+      <ul>
+        <a href="#box-one"><li class="left-end active">The roblem</li></a>
+        <a href="#key-factors" onclick="scroll(1)"><li>Key Factors</li></a>
+        <a href="#questions"><li>The Question</li></a>
+        <a href="#solution"><li class="right-end ">The Solution</li></a>
+      </ul>
+    </nav>
+
+    <div class="box" id="box-one">
+      <p id="main-thought">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.</p>
+      <p>Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.<sup>1</sup> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<sup>2</sup></p>
+
+      <div class="quote">
```

Comment:
Some semantic tag options here would be to use `<blockquote>` and `<cite>`, for quotes and citations respectively.